README.md

```diff
@@ -240,22 +240,527 @@ For complete documentation of the `mp-build` tool see the [MP Build Tool Documen
 
 ## Walkthroughs
 
-### Creating a New Management Pack (Cassandra-DB)
+### Creating a New Management Pack (Alibaba Cloud)
 <details><summary>
 This guide assumes you have already set up the SDK and know how to create a new project. 
-It walks you through the steps necessary to monitor an endpoint, using Cassandra DB as 
+It walks you through the steps necessary to monitor an endpoint, using Alibaba Cloud as 
 an example.</summary>
 TODO
 </details>
 
 ### Extending an Existing Management Pack (MySQL)
 <details><summary>
 This guide assumes you have already set up the SDK and know how to create a new project. 
-It walks you through the steps necessary to extend an existing Management Pact to add
+It walks you through the steps necessary to extend an existing Management Pack to add
 additional data, using the MySQL Management Pack as an example.</summary>
-TODO
-</details>
 
+Extending an existing management pack is similar to creating a new management pack, but 
```

Comment:
<img width="853" alt="image" src="https://user-images.githubusercontent.com/1360302/232781615-22daffb6-0bcc-4d02-bb7a-82fef3714965.png">

can we add an additional empty line for visual spacing?

Reply:
@kjrokos this one looks the same after updates:

<img width="860" alt="image" src="https://user-images.githubusercontent.com/1360302/232879929-1cc6832e-270f-464b-b54c-d22bc07975ae.png">

Reply:
seems to be an issue with GitHub markdown. Resolving as is